src/utils/feature.flag.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Feature flags will be determined by environment variables and all environment variables in nodejs are
+ * either string or undefined. This function will ensure that 'false', '0', 'off' etc remain falsy
+ */
+export const isActiveFeature = (flag: string | undefined): boolean => {
+
+  if (flag === undefined) {
+    return false;
+  }
+  const featureFlag = flag.toLowerCase();
```

Comment:
We have already set the feature flag to be `false`? I do not think we need all of these condition

Reply:
The relates to all feature flags not just this one.